addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -65,14 +65,6 @@ html[data-edit_translations="1"] {
     }
 }
 
-// SNIPPET PANEL
-$i: 1;
-@each $font-name, $font-config in $o-theme-font-configs {
-    we-toggler.o_we_option_font_#{$i}, we-button.o_we_option_font_#{$i} > div {
-        font-family: o-safe-get($font-config, 'family', $font-family-base);
-    }
-    $i: $i + 1;
-}
```

Comment:
In stable, let's mark this as deprecated, but don't remove it, we'll remove in master in the forward-ported version (just add "TODO remove in master"). I agree the risk of removing this is very small but it's just a good habit to keep to not suppose you can remove things in stable (especially dead code, it does not hurt anyone in stable versions).